roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -59,6 +59,16 @@
   failed_when: "'already exists' not in osmiq_perm_task.stderr and osmiq_perm_task.rc != 0"
   changed_when: osmiq_perm_task.rc == 0
 
+- name: Configure 3_2 role/user permissions
+  command: >
+    {{ openshift.common.admin_binary }} {{item}}
+    --config={{manage_iq_tmp_conf}}
+  with_items: "{{manage_iq_openshift_3_2_tasks}}"
+  register: osmiq_perm_3_2_task
+  failed_when: osmiq_perm_3_2_task.rc != 0
+  changed_when: osmiq_perm_3_2_task.rc == 0
```

Comment:
```oc get clusterpolicybindings system:image-auditor``` does not seem to work for me with
```Error from server: clusterpolicybinding "system:image-auditor" not found```

@deads2k  @abutcher is there currently a way to get this info?

https://trello.com/c/YqfYqKdS/545-8-policy-related-endpoints-need-work seems to be related